src/snowcli/cli/nativeapp/init.py

```diff
@@ -0,0 +1,291 @@
+                git checkout
+        """
+    try:
+        subprocess.run(
+            clone_command,
+            shell=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        )
+        subprocess.run(
+            sparse_checkout_command,
+            shell=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        )
+    except subprocess.CalledProcessError as err:
+        log.error(err.stderr)
+        raise GitCloneError()
+
+
+def _move_and_rename_project(
+    source_parent_directory: Path,
+    target_parent_directory: Path,
+    repo_sub_directory: str,
+    new_name: str,
+):
+    """
+    Move the newly cloned repository's sub directory from its source directory to target directory, and rename to new_name.
+
+    Args:
+        source_parent_directory (Path): The source parent directory of the sub directory.
+        target_parent_directory (Path): The target parent directory for the sub directory.
+        repo_sub_directory (str): The sub directory name within the cloned repository.
+        new_name (str): The new name to give to the sub directory after moving.
+
+    Returns:
+        None
+    """
+
+    # Move to target parent directory
+    source_path = source_parent_directory.joinpath(repo_sub_directory)
+    source_path.rename(target_parent_directory / source_path.name)
+
+    # Rename directory
+    old_name = target_parent_directory.joinpath(repo_sub_directory)
+    old_name.rename(old_name.parent / new_name)
+
+
+def render_snowflake_yml(parent_to_snowflake_yml: Path):
+    """
+    Create a snowflake.yml file from a jinja template at a given path.
+
+    Args:
+        parent_to_snowflake_yml (Path): The parent directory of snowflake.yml.jinja, and later snowflake.yml
+
+    Returns:
+        None
+    """
+
+    snowflake_yml_jinja = "snowflake.yml.jinja"
+
+    try:
+        generic_render_template(
+            template_path=parent_to_snowflake_yml.joinpath(snowflake_yml_jinja),
+            data={"project_name": parent_to_snowflake_yml.name},
+            output_file_path=parent_to_snowflake_yml.joinpath("snowflake.yml"),
+        )
+        os.remove(parent_to_snowflake_yml.joinpath(snowflake_yml_jinja))
+    except Exception as err:
+        log.error(err)
+        raise RenderingFromJinjaError(snowflake_yml_jinja)
+
+
+def render_nativeapp_readme(parent_to_readme: Path, project_name: str):
+    """
+    Create a README.yml file from a jinja template at a given path.
+
+    Args:
+        parent_to_readme (Path): The parent directory of README.md.jinja, and later README.md
+
+    Returns:
+        None
+    """
+
+    readme_jinja = "README.md.jinja"
+
+    default_application_name_prefix = clean_identifier(project_name)
+    default_application_name_suffix = clean_identifier(
+        get_env_username() or DEFAULT_USERNAME
+    )
+
+    try:
+        generic_render_template(
```

Comment:
This is a function from `render/commands.py`. 
I know that we already use logic of some commands in other commands, but we usually do it using "managers".
Maybe to be consistent, we should have `RenderManager`?
@sfc-gh-turbaszek WDYT?

Reply:
Personally I would move this function to common/utils module. This is something we may want to use for multiple usa-cases and keeping it command-specific does not make sense.